.gitignore

```diff
@@ -1,7 +1,23 @@
-development.log
-commands.log
+# Please do not add personal files
+
+# Logs
+*.log
+
+# Extras
 extra/lgit.rb
+extra/lgit.rb
```

Comment:
a few things are duplicated here by the looks of it

Reply:
Woops, forgot about that, also do you want me to just do ignore the whole extra folder?
And the notes could be replaced by *.note perhaps

Reply:
hmm I don't think I use the extra folder anymore so we can just remove that line completely.
An yeah using *.note sounds good to me :)